editor/editor_about.cpp

```diff
@@ -73,15 +87,22 @@ void EditorAbout::_version_button_pressed() {
 	DisplayServer::get_singleton()->clipboard_set(version_btn->get_meta(META_TEXT_TO_COPY));
 }
 
-void EditorAbout::_bind_methods() {
-	ClassDB::bind_method(D_METHOD("_version_button_pressed"), &EditorAbout::_version_button_pressed);
+void EditorAbout::_item_with_website_selected(int p_id, ItemList *p_il) {
+	const String website = String(p_il->get_item_metadata(p_id));
```

Comment:
```suggestion
	const String website = p_il->get_item_metadata(p_id);
```
No need for explicit constructing when assigning.